File: Long Contest 4/B_1_3_5.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int main() {
    int t;
    cin >> t;

    while (t--) {
        int n;
        cin >> n;
        int minOneBurle = n;

        for (int a = 0; a <= n / 5; ++a) {
            for (int b = 0; b <= n / 3; ++b) {
                int total = a * 5 + b * 3;
                if (total <= n) {
                    int remaining = n - total;
                    minOneBurle = min(minOneBurle, remaining);
                }
            }
        }

        cout << minOneBurle << endl;
    }

    return 0;
}

File: Long Contest 4/B_1_3_5.kt
fun main() {
    val t = readLine()!!.toInt()

    repeat(t) {
        val n = readLine()!!.toInt()
        var minOneBurle = n

        for (a in 0..(n / 5)) { 
            for (b in 0..(n / 3)) { 
                val total = a * 5 + b * 3
                if (total <= n) {
                    val remaining = n - total
                    minOneBurle = minOf(minOneBurle, remaining)
                }
            }
        }

        println(minOneBurle)
    }
}
